### ISSUE TITLE
[MAS4.1.2] Name/ Alt-text property is not proprly defined for Images.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1339]
**Actual Result:**
When focus lands on the Image then narrator is reading 'Group'.
Name property does not defined properly for 'Image'.

**Note -** Same issue repro at -
1. URL-> Help-> Show a simple  Adaptive cards button - Name property does not defined proerly for uploaded content after activate the Show a simple  Adaptive cards button button, narrator reading 'Group' while focus lands on the Content with help of the Tab key. Attachment " MAS4.1.2_Name for uploaded content"
2. URL-> Help-> Show an Adaptive cards with all types of inputs - New Uploaded data -> Show Cards button -> Images - Focus lands on the images while navigate in scan mode, narrator is reading 'Group'.
Attachment " MAS4.1.2_Name for Image".
3. Same issue repro on all decorative images, narrator is reading group while navigate in scan mode, Name property does not defined properly for decorative images.

**Expected Result:**
Name property should be defined properly for 'Image'.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Content will upload, Navigate to any "Show a rich message using Adaptive cards" button by using Tab key and press Enter on it.
5. New data will upload.
6. Open Narrator with Scan mode (Caps+Space key)
7. Navigate to the new uploaded content by using Up/Down arrow keys.
8. Observe that name property is not defined properly for images and narrator is reading "group".

**User Impact:**
If Name,State and Role property is not defined properly for controls then the users will get confuse about the purpose of that control.

**MAS Reference:**
MAS4.1.2 - [Name,Role,Value](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
1. [Usable_Name for the image does not defined properly.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2574901/Usable_Name.for.the.image.does.not.defined.properly.zip)
2. [MAS4.1.2_Name for uploaded content.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2575065/MAS4.1.2_Name.for.uploaded.content.zip)
3. [MAS4.1.2_Name for Image.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2579297/MAS4.1.2_Name.for.Image.zip)

**Comment by compulim** [comment]
There are two issues here.

#### Bug on Adaptive Cards

If we didn't define `alt` in the image, Adaptive Cards generated `<img alt="undefined">`. We repro-ed this bug on https://adaptivecards.io/samples/SportingEvent.html.

![image](https://user-images.githubusercontent.com/1622400/49117311-15ff6080-f255-11e8-88c4-608e213e237a.png)

#### Not able to repro

We used Narrator on Edge. We heard "image undefined", instead of "group". So we are not able to repro.

**Comment by corinagum** [comment]
Confirmed as external by accessibility team

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.1.2
